webpack/InsightsHostDetailsTab/InsightsTab.scss

```
@@ -88,3 +88,7 @@
 #new_host_details_insights_tab {
   padding: 24px;
 }
+
+#dropdown-container ul.pf-v5-c-menu__list {
```

The CSS may be specific now, but I'd prefer the name of the id to also be specific.

```suggestion
#total-risks-dropdown-container ul.pf-v5-c-menu__list {
```

---

Updated.